ibis/backends/bigquery/tests/system/udf/test_udf_execute.py

```diff
@@ -28,12 +28,8 @@ def df(alltypes):
 
 
 def test_udf(alltypes, df):
-    @udf(
-        input_type=[dt.double, dt.double],
-        output_type=dt.double,
-        determinism=True,
-    )
-    def my_add(a, b):
+    @udf.scalar.python(determinism=True)
```

Comment:
Not really a huge fan of this parameter name, but I kept it to match the previous interface.